3D-Engine/Model.cpp

```diff
@@ -0,0 +1,9 @@
+#include <string>
+#include "Model.h"
+
+void Model::Load(std::string path)
```

Comment:
This is copying the string, which is probably not what you want. const std::string& path is better